components/collection/drop/GenerativePreview.vue

```diff
@@ -116,7 +116,7 @@ const { imageDataPayload, imageDataLoaded } = useGenerativeIframeData()
 const { start: startTimer } = useTimeoutFn(() => {
   // quick fix: ensure that even if the completed event is not received, the loading state of the drop can be cleared
   // only applicable if the drop is old one that missing`kodahash/render/completed` event
-  if (!props.mintCountAvailable && !imageDataLoaded.value) {
+  if (!imageDataLoaded.value) {
     isLoading.value = false
     emit('generation:end')
   }
```

Comment:
It's better because it's a fallback if the loaded event does not trigger.
Maybe we should also change the comment.

Reply:
Updated a little bit. Feel free to close this PR in case it is not needed the changes 👍🏻